platform-hub-api/spec/factories/hash_records.rb

```diff
@@ -43,5 +43,16 @@
       end
     end
 
+    factory :feature_flags_hash_record do
+      scope 'general'
+      id 'feature_flags'
+      data do
+        {
+          'some-flag': false,
+          'other-flag': true
```

Comment:
Similar point here (and in the spec above) – may be good to use underscores to better represent how we'll actually have the feature flags.